===================================================================
RCS file: /cvs/openafs/src/pinstall/install.c,v
retrieving revision 1.21
retrieving revision 1.22
diff -u -r1.21 -r1.22
--- openafs/src/pinstall/install.c	2003/07/25 04:38:03	1.21
+++ openafs/src/pinstall/install.c	2003/08/08 21:00:09	1.22
@@ -50,7 +50,7 @@
 #include <afs/param.h>
 
 RCSID
-    ("$Header: /var/cvsroot/gentoo-x86/net-fs/openafs/files/openafs-pinstall-execve.patch,v 1.2 2004/07/18 03:54:09 dragonheart Exp $");
+    ("$Header: /var/cvsroot/gentoo-x86/net-fs/openafs/files/openafs-pinstall-execve.patch,v 1.2 2004/07/18 03:54:09 dragonheart Exp $");
 
 #include <stdio.h>
 #include <pwd.h>
@@ -183,6 +183,9 @@
 {
     int pid;
     pid_t status;
+    static char *env[] = { 
+	0,
+    };
     static char *strip[] = {
 	"strip", 0, 0,
     };
@@ -201,7 +204,7 @@
     case 0:			/* child        */
 	copy[1] = iname;
 	copy[2] = oname;
-	execve("/bin/cp", copy, NULL);
+	execve("/bin/cp", copy, env);
 	perror("/bin/cp");
 	exit(1);
 
@@ -246,7 +249,7 @@
 #else
 #define	STRIP_BIN	"/bin/strip"
 #endif
-	execve(STRIP_BIN, strip, NULL);
+	execve(STRIP_BIN, strip, env);
 	perror(STRIP_BIN);
 	exit(1);
 
@@ -272,6 +275,9 @@
 quickStrip(char *iname, char *oname, int ignored, int copy_only)
 {
     int pid, status;
+    static char *env[] = { 
+	0,
+    };
     static char *strip[] = {
 	"strip", 0, 0,
     };
@@ -290,7 +296,7 @@
     case 0:			/* child        */
 	copy[1] = iname;
 	copy[2] = oname;
-	execve("/bin/cp", copy, 0);
+	execve("/bin/cp", copy, env);
 	perror("/bin/cp");
 	exit(1);
 
@@ -328,7 +334,7 @@
 
     case 0:			/* child        */
 	strip[1] = oname;
-	execve("/bin/strip", strip, 0);
+	execve("/bin/strip", strip, env);
 	perror("/bin/strip");
 	exit(1);
 
